pages/StartView.neu.page.ts

```diff
@@ -0,0 +1,32 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+export class StartViewPage {
+  /* add global locators here */
+  readonly page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  /* actions */
+  public async waitForPageLoad(): Promise<void> {
+    await this.page.getByTestId('start-card-headline').waitFor({ state: 'visible' });
+    await expect(this.page.getByTestId('start-card-headline')).toHaveText('Startseite');
+    await this.serviceProvidersAreLoaded();
+  }
+
+  /* assertions */
+  private async serviceProvidersAreLoaded(): Promise<void> {
+    await this.page.waitForResponse((response) => response.url().includes('/api/provider') && response.status() === 200);
+    await this.page.waitForResponse('/api/provider/**/logo');
```

Comment:
```suggestion
await Promise.all([
  this.page.waitForResponse((response) => response.url().includes('/api/provider') && response.status() === 200),
  this.page.waitForResponse('/api/provider/**/logo')
]);
```
Do we need the first request to fetch the logos? If yes then ignore this suggestion.

Reply:
I think so, it always comes first. I'll leave this suggestion out